src/search-manager/SearchManager.ts

```diff
@@ -44,6 +44,8 @@ export interface SearchContextData {
   hasError: boolean;
   collectionHits: CollectionHit[];
   totalCollectionHits: number;
+  contentAndCollectionHits: (ContentHit | CollectionHit)[];
+  totalContentAndCollectionHits: number;
```

Comment:
I would strongly prefer that we simplify this to have just one `hits: (ContentHit | CollectionHit)[];` property instead of having `hits`, `collectionHits`, and `contentAndCollectionHits`

Reply:
will do it that way. I was hoping to avoid breaking stuff but it seems that the way you described will break less things.

Reply:
The code (and UX) has been getting complicated so it's more important to me that we simplify the code, even if it results in a bigger PR.